doc/reference/reports.rst

```diff
@@ -282,24 +282,25 @@ named :samp:`report.{module.report_name}`. If it exists, it will use it to
 call the QWeb engine; otherwise a generic function will be used. If you wish
 to customize your reports by including more things in the template (like
 records of others models, for example), you can define this model, overwrite
-the function ``render_html`` and pass objects in the ``docargs`` dictionary:
+the function ``get_report_values`` and pass objects in the ``docargs`` dictionary:
 
 .. code-block:: python
 
     from odoo import api, models
 
     class ParticularReport(models.AbstractModel):
         _name = 'report.module.report_name'
+
         @api.model
-        def render_html(self, docids, data=None):
+        def get_report_values(self, docids, data=None):
             report_obj = self.env['report']
```

Comment:
Just a little last thing, could you change this to 
`self.env['ir.actions.report']` which is the correct model name in 11 (dunno since when, but self.env['report'] isn't valid as I see in the overrides of get_report_values in the codebase.

Reply:
oh, right, what an oversight haha